packages/trpc/package.json

```diff
@@ -5,6 +5,8 @@
   "types": "./index.ts",
   "license": "MIT",
   "scripts": {
+    "lint": "eslint . --ext .ts,.tsx",
```

Comment:
Do we actually require the extension for all these scripts? 🤔

Reply:
It's optional; I added this because we can run the script from the root of the repo and lint it.

Reply:
Not required I think 🤔, but it might help the linter to look only for TypeScript files.

Reply:
I reckon I'd let it do it's default action:
`eslint --fix .`

Reply:
Done ✅